### ISSUE TITLE
failed to reproduce the accuracy

### ISSUE THREAD

**Issue by qizhust** [kexinyi/ns-vqa#16]
Hi, I retrain the model but get lower accuracy (overall_acc=0.72 for pretrained parser and 0.88 for reinforce-finetuning), do you have any suggestions?
